test/functional/test_framework/util.py

```diff
@@ -7,6 +7,7 @@
 from base64 import b64encode
 from binascii import hexlify, unhexlify
 from decimal import Decimal, ROUND_DOWN
+from subprocess import CalledProcessError
```

Comment:
nit: my personal preference is for imports to be sorted by module name (ie `from subprocess...` should be between `import re` and `import time`)

Reply:
My bad, that's also my preference.